src/scenekit.cs

```diff
@@ -152,7 +155,14 @@ interface SCNAnimatable {
 #if XAMCORE_2_0
 		[Abstract]
 #endif
-		[Mac (10,9)]
+		[Introduced (PlatformName.WatchOS, 3, 0)]
+		[Deprecated (PlatformName.WatchOS, 4, 0, message: "Use SCNAnimationPlayer.Paused] instead")]
```

Comment:
The message seems to have an unwanted typo `]` also I think our tests will complain about ending dot. You might want to use

> Use 'SCNAnimationPlayer.Paused' instead.